### ISSUE TITLE
Failure in base capture event

### ISSUE THREAD

**Issue by Winston2108** [ModernColdWar/RedStormRising#150]
Failed to capture Gelendzhik by red last night.  Base showed red on the F10 map, but no red defences spawned and RSR status marker wasn't updated to red.

Checked in Game Master slot that no blue units were nearby.

```
2020-02-09 23:15:40.715 INFO    SCRIPTING: BaseCapturedHandler|EventFunction|17: Base captured event for Gelendzhik captured by $2
2020-02-09 23:15:40.715 INFO    SCRIPTING: Error in SCHEDULER function:[string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\state.lua"]:181: bad argument #1 to 'insert' (table expected, got nil)
2020-02-09 23:15:40.715 INFO    SCRIPTING: stack traceback:
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:6740: in function <[string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:6736>
        [C]: in function 'insert'
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\state.lua"]:181: in function 'setOwner'
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\state.lua"]:190: in function 'setBaseOwner'
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\baseCapturedHandler.lua"]:18: in function 'EventFunction'
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:7053: in function <[string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:7052>
        [C]: in function 'xpcall'
        [string "C:\Users\Administrator\Saved Games\DCS.openbeta_server\Scripts\RSR\Moose.lua"]:7051: in function 'onEvent'
        [string "Scripts/World/EventHandlers.lua"]:13: in function <[string "Scripts/World/EventHandlers.lua"]:11>
2020-02-09 23:15:40.715 INFO    Scripting: event:place=Gelendzhik,t=62113.86,type=base captured,placeDisplayName=Gelendzhik,
```

**Comment by Winston2108** [comment]
Should be fixed in 846c370c2a48d0a7924f10a3ab0e0381487a3998

**Comment by madrabbit711** [comment]
RSR 3.0 old code.  Closing issue.
